src/context/AuthContext/AuthContext.js

```diff
@@ -0,0 +1,42 @@
+import { createContext, useContext, useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { loginHandlerService } from "../../service/authentication/loginHandlerService";
+
+const AuthContext = createContext({});
+
+const AuthProvider = ({ children }) => {
+  const authToken = JSON.parse(localStorage.getItem("AUTH-TOKEN"));
+
+  const navigate = useNavigate();
+  const location = useLocation();
+  console.log(location);
```

Comment:
Remove Console if no need

Reply:
It is removed in the local file, don't know why it didn't update in github